plugins/extraction/table.mli

```diff
@@ -46,7 +46,7 @@ val info_file : string -> unit
 (*s utilities about [module_path] and [kernel_names] and [GlobRef.t] *)
 
 val occur_kn_in_ref : MutInd.t -> GlobRef.t -> bool
-val repr_of_r : GlobRef.t -> ModPath.t * Label.t
+val repr_of_r : GlobRef.t -> KerName.t
```

Comment:
Is there a risk of plugins linking to extraction / using this function or should it be considered an internal API? I need to decide whether it is safe to backport this PR.

Reply:
(Too) Late answer: I have no idea.  Not backporting should be ok also since 8.18 is also on rails.

Reply:
Indeed, this is too late. I had forgotten about this risk and actually ended up backporting, so this is shipped in 8.17.1.